pom.xml

```diff
@@ -521,7 +521,7 @@
                   </manifestEntries>
                 </transformer>
                 <transformer implementation="org.apache.maven.plugins.shade.resource.IncludeResourceTransformer">
-                  <file>log4j2.xml</file>
+                  <file>log4j2-test.xml</file>
```

Comment:
This suggests that the test configuration should be included in the production artefacts, is that correct?

Reply:
Sorry, I must have gotten confused and changed it in the wrong place.  Try it now.